### DISCUSSION TITLE
The app is WAY TOO NICE

### DISCUSSION THREAD

**Discussion by IdkWhatToCallMe123** [character-map-uwp/Character-Map-UWP#254] [General]
The app is so GOOD that it causes several problems for me:

- You can't leave a worthy review on the Microsoft Store since it doesn't support 6 star reviews
- I spend way to much time playing around with all the buttons instead of actually using it
- I find my self constantly restarting the app to see that BEAUTIFUL splash animation

pls make a slightly crappier version so i dont waste more time

**Comment by JohnnyWestlake** [comment]
Apologies to hear about your problems, I can recommend a couple of solutions for you:

Go to `Settings -> Look & Feel -> Enable Animations` to Off. And for good measure, change Design Style in the same screen to `Classic Windows` and restart the app. Now your general productivity will increase 👌

And avoid pressing the button to switch between Type Ramp and Character Map View 🤔

**Comment by IdkWhatToCallMe123** [comment]
Wait you said WHAT BUTTON

hang on brb
